### ISSUE TITLE
noplot option in "recovery"

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#227]
In GitLab by @obellprat on Dec 21, 2016, 11:39

Hi, 

it would be useful to have the -np option (no plot) also in the recovery function since the plot function is  slow for large experiments.

@dmanubens @ jlope2

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Dec 21, 2016, 11:51

Hi @obellprat,

What version are you using?

It should be there!

**Comment by kinow** [comment]
In GitLab by @obellprat on Dec 21, 2016, 12:33

I'm using version 3.6.0, it seems to be the defaul version:
```
obellpra@ bscearth315:~$ autosubmit -v
3.6.0

obellpra@ bscearth315:~$ autosubmit recovery -h
usage: autosubmit recovery [-h] [-all] [-s] [--hide] expid

recover specified experiment

positional arguments:
  expid       experiment identifier

optional arguments:
  -h, --help  show this help message and exit
  -all        Get completed files to synchronize pkl
  -s, --save  Save changes to disk
  --hide      hides plot window
obellpra@ bscearth315:~$ autosubmit setstatus -h
usage: autosubmit setstatus [-h] [-np] [-s] -t
                            {READY,COMPLETED,WAITING,SUSPENDED,FAILED,UNKNOWN,QUEUING,RUNNING}
                            (-fl LIST | -fc FILTER_CHUNKS | -fs {Any,READY,COMPLETED,WAITING,SUSPENDED,FAILED,UNKNOWN} | -ft FILTER_TYPE)
                            [--hide]
                            expid
```

**Comment by kinow** [comment]
In GitLab by @ jlope2 on Dec 21, 2016, 13:07

This feature it's already available since Autosubmit 3.7.0.

You can use Autosubmit 3.7.6 which is quite stable.
